packages/waku/src/router/common.ts

```diff
@@ -1,5 +1,5 @@
-export type RouteProps = {
-  path: string;
+export type RouteProps<Path extends string = string> = {
+  path: Path;
   query: string;
   hash: string;
```

Comment:
@dai-shi would `hash: string | undefined` be correct here since these could be used on the server?

Or should undefined only be added to hash for `PageProps`?

Reply:
`RouteProps` defined in common.ts is for both server & client.
`hash` is added afterwards, which only exists on the client.
So, there's already a mismatch, which should be improved.
For now, we should omit `hash` for `PageProps` (not `undefined`).

Reply:
wait, i think that's my misunderstanding.
`hash` on the server is always `''`, so the type is correct.
we don't need to omit `hash`.

Can you confirm it?

Reply:
So when running the dev server with examples/10_fs-router the hash is not included on the client or the server


```
// from console.log(JSON.stringify(props))
{"path":"/","query":"","children":{"key":null,"props":{},"_owner":null,"_store":{}}}
```

Reply:
changing the page to `render: dynamic` had no impact either... Is this a bug?

Reply:
thanks for confirming. sounds like i need to double check the source code. I'm probably confused with something else (it's probably client code runs on the server in SSR.)

So, I think my original comment is correct at least for now. 👇 


> For now, we should omit `hash` for `PageProps` (not `undefined`).

Reply:
sounds good, omitted 👍